igvc_navigation/src/mapper/main.cpp

```diff
@@ -1,41 +1,31 @@
+// Subscribes to Point Cloud Data, updates the occupancy grid, then publishes the data.
+
 #include <cv_bridge/cv_bridge.h>
 #include <igvc_msgs/map.h>
-#include <math.h>
 #include <nav_msgs/Odometry.h>
 #include <pcl/point_cloud.h>
 #include <pcl_ros/point_cloud.h>
 #include <pcl_ros/transforms.h>
-#include <ros/publisher.h>
-#include <ros/ros.h>
-#include <sensor_msgs/Image.h>
-#include <stdlib.h>
-#include <Eigen/Core>
-#include <opencv2/core/eigen.hpp>
-#include <opencv2/opencv.hpp>
-#include "tf/transform_datatypes.h"
 #include <igvc_utils/RobotState.hpp>
 
-igvc_msgs::map msgBoi;  // >> message to be sent
 cv_bridge::CvImage img_bridge;
-sensor_msgs::Image imageBoi;  // >> image in the message
 
 ros::Publisher map_pub;
```

Comment:
comment what each of these publishers are for, it's not immediately clear what the difference is